flixel/FlxCamera.hx

```diff
@@ -1703,8 +1707,11 @@ class FlxCamera extends FlxBasic
 			if (FxAlpha == 0)
 				return;
 
-			var targetGraphics:Graphics = (graphics == null) ? canvas.graphics : graphics;
+			final targetGraphics = (graphics == null) ? canvas.graphics : graphics;
 
+			#if (openfl > "8.7.0")
+			targetGraphics.overrideBlendMode(null);
+			#end
```

Comment:
At a glance this seems like the only change related to the titular bug, and the rest seem like unrelated style changes, is that the case?

Reply:
Yes.

The style change for drawFX() was made to make it more readable, but if it was too unnecessary - I don't see a problem in reverting it and leaving only the bugfix.